aapcs64/aapcs64.rst

```diff
@@ -1468,10 +1613,74 @@ Table 7: Short vector extended types
 | __Bfloat16x8\_t | bfloat16x8\_t     | half-precison Brain float| 8         |
 +-----------------+-------------------+--------------------------+-----------+
 
-C++ Mangling
-------------
+APPENDIX Support for Scalable Vectors
+=====================================
+
+As an optional extension, the AArch64 architecture supports a number of
+scalable-vector operations. To facilitate accessing these operations
+from C and C++ a number of extended types need to be added to the
+language.
+
+Following the conventions used for adding types to C99, and the
+conventions established in `APPENDIX Support for Advanced SIMD Extensions`_,
+a number of additional types (internal types) are defined
+unconditionally. To facilitate use in applications a header file
+``arm_sve.h`` is also defined that maps these internal types onto more
```

Comment:
Also what? I'd remove the also.
```suggestion
``arm_sve.h`` is defined that maps these internal types onto more
```

Reply:
OK.  This too is a carry-over from the Advanced SIMD stuff.